dev/docker/python/Dockerfile

```diff
@@ -1,4 +1,8 @@
 # start with the `delphi_python` image
 FROM delphi_python
 
+# copy over all source files
+COPY repos repos
+RUN chmod -R o+r repos/
```

Comment:
Hey @sgratzl I added this to the Dockerfile to avoid having to repeatedly rebuild both `delphi_python` and `delphi_web_python` during development. Will this break CI or can I leave it in?

Reply:
why don't you just add this as a volume into the docker container? then you just have to restart the container. At least that is what I'm doing.